bin/prepare-native.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
+sudo apt -y install openssl libssl-dev libsdl2-dev libulfius-dev liborcania-dev
```

Comment:
This needs to be included in the CI builder scripts on github, or the builds will fail.

Reply:
Yes sorry didn't looked into the build pipline of Github. Then I first have to prepare a change for the build pipeline I guess. I think for native linux build the .github/actions/setup-base/action.yml would be the best place?

Reply:
That's something we can do afterwards. i commented more like a todo item :-)

the runners are always picked up from master, you can't fix that here

Reply:
Adding these packages to the Pi builder now.